.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Can you please share why we need this configuration? 🙏

Reply:
Not needed. Came as a result of me using a newer yarn locally.